src/cores/auth/useLoginForm.jsx

```diff
@@ -0,0 +1,28 @@
+
+import { useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import useForm from '../../components/useForm';
+import { login } from '../../features/auth/authSlice';
+import * as authServices from '../../services/auth.services';
+
+const useLoginForm = () => {
+
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const { inputValues, handleChange } = useForm();
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      const user = await authServices.login(inputValues);
+      dispatch(login(user));
+      navigate("/dashboard");
+    } catch(e) {
+      alert(e)
```

Comment:
```suggestion
      alert(e)
```

Las alertas son super feas, lo mejor seria que hiciesen un modal dedicado a mostrar las alertas de error o mostraran un toastr

Reply:
no

Reply:
Usen toastr son bien sencillos de usar

Reply:
jamas